media_driver/agnostic/gen11/codec/hal/codechal_vdenc_avc_g11.cpp

```diff
@@ -692,6 +688,7 @@ CodechalVdencAvcStateG11::CodechalVdencAvcStateG11(
     Mos_SetVirtualEngineSupported(m_osInterface, true);
 
     m_vdboxOneDefaultUsed = true;
+    m_openCommonKernel = true;
```

Comment:
I REALLY don't understand what you are doing with this flag!

Reply:
it's the same purpose as the "FREE_KERNEL" MACRO proposed, however, with this flag I can control by each platform

Reply:
I posted big comment above. Shortly: how you name the flag and how you use it in the code is _misleading_. "common" is misleading, "kernel" instead of "kernel>>>s<<<" is misleading. And having if (m_openCommonKernel) instead of if ( >>>!<<< m_openCommonKernel) is misleading.

IMHO, why you cover code which should be executed for both cases when you have open source kernels and when you have closed source kernels?! This is illogical. Expectation is that you will strip out the code which should _not_ be executed for one of the path, i.e. the code for closed source part.